src/ui/Snackbar/SnackbarList.tsx

```diff
@@ -0,0 +1,49 @@
+import { AnimatePresence, motion, Variants } from 'framer-motion';
+import { ComponentProps } from 'react';
+import Snackbar from './Snackbar';
+
+type TSnackbar = ComponentProps<typeof Snackbar> & {
+  id: string;
+};
+type Props = {
+  className?: string;
+  snacks: TSnackbar[];
+};
+function SnackbarList({ className, ...props }: Props) {
```

Comment:
Let's call this  `SnackbarGroup` as we called `ModalGroup`. Rename the file as well.

Reply:
So things like `Cards` is `CardGroup` as well?

Reply:
If we have such a thing, yea,. that sounds good